CONTRIBUTING.md

```diff
@@ -31,41 +92,80 @@ The list of labels for current issues are the same as for the BIDS [here](https:
+A member of the BIDS Starter Kit team will review your changes to confirm that they can be merged into the main codebase.
 
+A [review][github-review] will probably consist of a few questions to help clarify the work you've done. Keep an eye on your github notifications and be prepared to join in that conversation.
+
+You can update your [fork][github-fork] of the BIDS Starter Kit [repository][bids-starterkit-repo] and the pull request will automatically update with those changes. You don't need to submit a new pull request when you make a change in response to a review.
+
+GitHub has a [nice introduction][github-flow] to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions :balloon:.
+
+
+## Recognizing contributions
+
+BIDS follows the [all-contributors][all-contributors] specification, so we welcome and recognize all contributions from documentation to testing to code development. You can see a list of current contributors in the [BIDS specification][bids-specification].
 
 ## Thank you!
 
 You're awesome. :wave::smiley:
 
 <br>
 
-*&mdash; Based on contributing guidelines from the [STEMMRoleModels](https://github.com/KirstieJane/STEMMRoleModels) project.*
-
-
+*&mdash; Based on contributing guidelines from the [STEMMRoleModels][stemm-role-models] project.*
+
+[all-contributors]: https://github.com/kentcdodds/all-contributors#emoji-key
+[bids]: http://bids.neuroimaging.io
+[bids-specification]: https://docs.google.com/document/d/1HFUkAEE-pB-angVcYe6pf_-fVf4sCpOHKesUvfb8Grc/edit#heading=h.hds2i7ii7hjo
+[bids-mailinglist]: https://groups.google.com/forum/#!forum/bids-discussion
+[bids-starterkit-issues]: https://github.com/INCF/bids-starter-kit/issues
+[bids-starterkit-labels]: https://github.com/INCF/bids-starter-kit/labels
+[bids-starterkit-repo]: https://github.com/INCF/bids-starter-kit
+[bids-starterkit-wiki]: https://github.com/INCF/bids-starter-kit/wiki
+[brainhack-slack-starterkit]: https://brainhack.slack.com/messages/C8RG7F6PN
+[brainhack-slack-invite]: https://brainhack-slack-invite.herokuapp.com
+[intro-github-wiki]: https://help.github.com/articles/about-github-wikis
+[dont-push-pull-request]: https://www.igvita.com/2011/12/19/dont-push-your-pull-requests
+[git]: https://git-scm.com
+[github]: https://github.com
+[github-branches]: https://help.github.com/articles/creating-and-deleting-branches-within-your-repository
+[github-fork]: https://help.github.com/articles/fork-a-repo
+[github-flow]: https://guides.github.com/introduction/flow
+[github-mergeconflicts]: https://help.github.com/articles/about-merge-conflicts
+[github-pullrequest]: https://help.github.com/articles/creating-a-pull-request
+[github-review]: https://help.github.com/articles/about-pull-request-reviews
+[github-syncfork]: https://help.github.com/articles/syncing-a-fork
+[gitter]: https://gitter.im/INCF/bids-starter-kit
+[gsoc]: https://summerofcode.withgoogle.com
+[jerry-maguire]: http://gph.is/1ULS704
```

Comment:
This is hysterical.

Reply:
:joy: :joy: Summarizes the message well!
Would it be slightly more direct to link to this? https://media.giphy.com/media/uRb2p09vY8lEs/giphy.gif 
(i.e. pure image source vs the giphy page with other posts/Ads/etc visible)

Reply:
Thanks @emdupre & @Park-Patrick for sanity checking my hilarious sense of humour 😉 

@Park-Patrick - sure - happy to link to the image itself.

Reply:
Updated the link ✔️